src/SqlExecutionArea.h

```diff
@@ -32,7 +34,8 @@ class SqlExecutionArea : public QWidget
     SqliteTableModel* getModel() { return model; }
     QTextEdit* getResultView();
     SqlTextEdit* getEditor();
-
+	ExtendedTableWidget *getTableResult();
```

Comment:
Hmmm, this seems to be using embedded tabs instead of spaces.

Would you be ok to fix that? :smile: